tests/src/session.spec.ts

```diff
@@ -262,4 +262,28 @@ describe('protocol', () => {
       expect(j.value).to.equal('4');
     });
   });
+
+  describe('#loadedSources', () => {
+    it('should *not* retrieve the list of loaded sources', async () => {
+      // `loadedSources` is not supported at the moment "unknown command"
+      const reply = await debugSession.sendRequest('loadedSources', {});
+      expect(reply.success).to.be.false;
+    });
+  });
```

Comment:
Leaving it in the tests so we know `loadedSources` is not supported.

Also fine to remove it if needed.

Reply:
I think it's a good thing to keep it.